entries/t/twingate.com.json

```
@@ -0,0 +1,17 @@
+{
+  "Twingate": {
+    "domain": "twingate.com",
+    "tfa": [
+      "totp",
+      "custom-hardware"
+    ],
+    "custom-hardware": [
+      "Biometrics",
+      "Yubico OTP"
+    ],
+    "documentation": "https://www.twingate.com/docs/two-factor-authentication-security-policies",
+    "categories": [
+      "remote"
```

I think security may be a more appropriate category. Or perhaps both
```suggestion
      "security"
```

---

I feel a little confused about the classification here, but I personally prefer remote. ❤️ @Carlgo11

---

Okay! Let's do remote then.